### PR TITLE
Converted to markup

### PR DESCRIPTION
**Pull request by CrudeRags**
Check it out

### CODE REVIEW

**Comment by TinasheMzondiwa** [comment]
Thank you for this @CrudeRags 
Right now everything is being formatted as a heading.
We're not correctly putting the new lines.

See below content for hymn 1 in english.json

# 1 Watchman Blow The Gospel Trumpet.\n\nWatchman, blow the gospel trumpet, Evry soul a warning give; Whosoever hears the message May repent, and turn, and live. CHORUSBlow the trumpet, trusty watchman,Blow it loud o\u2019er land and sea\u2026\u2026.(loud o\u2019er land and sea)God commissions, sound the message!Ev\u2019ry captive may be free. Sound it loud o\u2019er ev\u2019ry hilltop,Gloomy shade, and sunny plain;Ocean depths repeat the message,full salvation\u2019s glad refrain. Sound it in the hedge and highway, Earth\u2019s dark sports where exiles roam; Let it tell all things are ready, Father waits to welcome home. Sound it for the heavy laden,Weary, longing to be free.Sound a Saviour\u2019s invitation,Sweetly saying, \u201cCome to me\u201d.

**Comment by TinasheMzondiwa** [comment]
> I should add a space?

Yes, I am trying to figure how we can add new lines.
We do need a space after the word "CHORUS"

**Comment by CrudeRags** [comment]
Check it out now

File: New_Files/english.json [hunk 1]
```diff
@@ -0,0 +1,1502 @@
+[
+ {
+  "title": "1 Watchman Blow The Gospel Trumpet.",
+  "number": 1,
+  "content": "# 1 Watchman Blow The Gospel Trumpet.\n\n Watchman, blow the gospel trumpet,         Evry  soul a warning give;        Whosoever hears the message         May repent, and turn, and live.        CHORUSBlow the trumpet, trusty watchman,Blow it loud o\u2019er land and sea\u2026\u2026.(loud o\u2019er land  and sea)God commissions, sound the message!Ev\u2019ry captive may be free.        Sound it loud o\u2019er ev\u2019ry hilltop,Gloomy shade, and sunny plain;Ocean depths repeat the message,full salvation\u2019s glad refrain.        Sound it in the hedge and highway,  Earth\u2019s dark sports where exiles roam;        Let it tell all things are ready,         Father waits to welcome home.        Sound it for the heavy laden,Weary, longing to be free.Sound a Saviour\u2019s  invitation,Sweetly saying, \u201cCome to me\u201d.        "
```
Comment (TinasheMzondiwa) [inline New_Files/english.json:5]: We're missing `\n`s at the end here

**Review by TinasheMzondiwa — APPROVED** [review]

**Pull request merged**